src/queries.js

```diff
@@ -205,9 +217,15 @@ function getAllByLabelText(container, text, ...rest) {
   if (!els.length) {
     const labels = queryAllLabelsByText(container, text, ...rest)
     if (labels.length) {
-      throw getElementError(`Found a label with the text of: ${text}, however no form control was found associated to that label. Make sure you're using the "for" attribute or "aria-labelledby" attribute correctly.`, container)
+      throw getElementError(
+        `Found a label with the text of: ${text}, however no form control was found associated to that label. Make sure you're using the "for" attribute or "aria-labelledby" attribute correctly.`,
+        container,
+      )
     } else {
-      throw getElementError(`Unable to find a label with the text of: ${text}`, container)
+      throw getElementError(
+        `Unable to find a label with the text of: ${text}`,
```

Comment:
In other messages it's "with the text:" but here it's "with the text of:" — inconsistent.

Reply:
@sompylasar  should I act on this?

Reply:
Not sure, I just noticed.

Reply:
Don't bother. We can fix that in another PR if we want.